l10n_ch_payment_slip/wizard/bvr_import.py

```diff
@@ -197,16 +197,18 @@ def _prepare_line_vals(self, statement, record):
                   'type': (record['amount'] >= 0 and 'customer') or 'supplier',
                   'statement_id': statement.id,
                   }
-        line_ids = move_line_obj.search(
+        line = move_line_obj.search(
             [('transaction_ref', '=', reference),
              ('reconcile_id', '=', False),
              ('account_id.type', 'in', ['receivable', 'payable']),
              ('journal_id.type', '=', 'sale')],
             order='date desc',
         )
-        if line_ids:
+        if len(line) > 1:
+            raise exceptions.Warning(
+                "Too many credit lines for reference %s" % reference)
```

Comment:
We should replace credit with receivable/payable in order not to get confused with the credit line of credit control